docs/common/craft-parts/reference/plugins/ant_plugin.rst

```diff
@@ -0,0 +1,90 @@
+.. _craft_parts_ant_plugin:
+
+Ant plugin
+==========
+
+The Ant plugin builds Java projects using the `Apache Ant`_ build tool.
+
+After a successful build, this plugin will:
+
+* Create ``bin/`` and ``jar/`` directories in ``$CRAFT_PART_INSTALL``.
+* Find the ``java`` executable provided by the part and link it as
+  ``$CRAFT_PART_INSTALL/bin/java``.
+* Hard link the ``.jar`` files generated in ``$CRAFT_PART_SOURCE`` to
+  ``$CRAFT_PART_INSTALL/jar``.
```

Comment:
So the plugin actually builds in `$CRAFT_PART_SOURCE` instead of `$CRAFT_PART_BUILD`?

Reply:
thanks for catching this! Turns out this was wrong (it's `$CRAFT_PART_BUILD` as you say) because I copied this over from the maven plugin docs, which got this from a docstring. Fixed all three places.